src/WebJobs.Script/Workers/Rpc/RpcWorkerProcess.cs

```diff
@@ -50,11 +50,8 @@ internal override Process CreateWorkerProcess()
         internal override void HandleWorkerProcessExitError(WorkerProcessExitException langExc)
         {
             // The subscriber of WorkerErrorEvent is expected to Dispose() the errored channel
-            if (langExc != null && langExc.ExitCode != -1)
-            {
-                _workerProcessLogger.LogDebug(langExc, $"Language Worker Process exited.", _workerProcessArguments.ExecutablePath);
-                _eventManager.Publish(new WorkerErrorEvent(_runtime, _workerId, langExc));
-            }
+            _workerProcessLogger.LogDebug(langExc, $"Language Worker Process exited. Pid={langExc.Pid}.", _workerProcessArguments.ExecutablePath);
```

Comment:
Are we always sure that we have langExc value?

Reply:
Add validation here as well.